civihr_default_theme/scss/components/_chr-grid.scss

```diff
@@ -0,0 +1,104 @@
+/*
+util classes
+*/
+.chr-separator {
+    background-color: rgb(237, 237, 237);
+    height: 2px;
+}
+
+.chr-table__borderless tbody tr td,
+.chr-table__borderless tbody tr th,
+.chr-table__borderless thead tr th,
+.chr-table__borderless thead tr td,
+.chr-table__borderless tfoot tr th,
+.chr-table__borderless tfoot tr td {
+    border: none;
+}
+
+.chr-container {
+    margin-top: 60px;
+    /*background-color: rgb(247, 247, 249);*/
+}
+/*
+.chr-container__title {
+
+}
+*/
+.chr-container__row {
+  background-color: #f7f7f9;
```

Comment:
Are you using a colour that is defined in the *_variables.scss* partials? We should always strive to have a very small colour palette that we constantly reuse, instead of adding new ad-hoc colours